PowershellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingService.cs

```diff
@@ -279,11 +280,20 @@ public int GetPSBreakpointId(PowershellBreakpoint bp)
         /// <summary>
         /// Get runspace availability
         /// </summary>
+        /// <param name="executionPriority">Within execution priority, runspace will try to kill existing low priority task to free up runspace</param>
         /// <returns>runspace availability enum</returns>
-        public RunspaceAvailability GetRunspaceAvailability()
+        public RunspaceAvailability GetRunspaceAvailability(bool executionPriority)

```

Comment:
WCF operation contract does not support default value. I can give a wrapper method for it.